packages/synapse-react-client/src/components/MoreResources/MoreResources.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Stack, Typography, useTheme, Link } from '@mui/material'
+import { Link as RouterLink } from 'react-router'
+
+type ResourceConfig = {
+  title: string
+  description: string
+  link: string
+}
+
+export type MoreResourcesProps = {
+  resources: ResourceConfig[]
+}
+
+const MoreResources = ({ resources }: MoreResourcesProps) => {
+  const theme = useTheme()
+  return (
+    <Box
+      sx={{
+        padding: { xs: '40px', lg: '60px 80px 80px 80px' },
+        background: `linear-gradient(0deg, rgba(255, 255, 255, 0.90) 0%, rgba(255, 255, 255, 0.90) 100%), ${theme.palette.primary.main}`,
```

Comment:
maybe I'm going crazy but I feel like this is just a flat color when I look at it in Figma or Storybook. If it is, can we just use the flat color?